index.html

```diff
@@ -565,11 +565,49 @@ <h3>Authorization</h3>
 require conforming <a>VC API clients</a> to utilize secure authorization
 technologies when performing certain types of requests. Each HTTP endpoint
 defined in this document specifies whether or not authorization is required
-when performing a request.
+when performing a request. With the exception of the class of forbidden 
+authorization protocols discussed later in this section, the VC API is authorization 
+mechanism agnostic.
       </p>
       <p>
-This section details the authorization technologies that have been contemplated
-for use by conforming implementations. Other equivalent authorization
+The VC API is meant to be generic and useful in many scenarios that require
+the issuance, possession, presentation, and/or verification of Verifiable
+Credentials. To this end, implementers should consider the following
+classifications of use cases:
+      </p>
+      <ul>
+        <li>
+<i>Public</i>. A Public API is one that can be called with no authorization.
+Examples include an open witness or timestamp service (a trusted service that
+can digitally sign a message with a timestamp for an audit trail purpose), or
+an open retail coupon endpoint ("15% off cheeseburgers"). Public verifiers
+may exist as well, to act as an agnostic third party in a trust scenario.
+      </li>
+      <li>
```

Comment:
We should say something about capability-based URLs being a viable strategy for controlling access to and endpoint.

Reply:
@dlongley said, try this: "to, for example, have an access control token, or capability URL, or invoke a capability"